### ISSUE TITLE
replatform: missing aria-label

### ISSUE THREAD

**Issue by cantsin** [18F/18f.gsa.gov#4041]
Example:

```
--- original/work-with-us/index.html	2024-08-06 08:24:30.365013882 -0700
+++ changed/work-with-us/index.html	2024-08-06 08:32:33.406031045 -0700
```

```diff
-      <div aria-label="Agency description" class="usa-identifier__identity text-base-lightest">
+      <div class="usa-identifier__identity text-base-lightest">
        <p class="usa-identifier__identity-domain">
         18f.gsa.gov
        </p>
```

**Comment by beechnut** [comment]
When we brought over the templates from 18F/guides, this "usa identifier" footer came with it. I suspect that this change came from using the guides' templates.

We can see in this screenshot that the guide name does not have the alt attribute:

![Screenshot 2024-08-07 at 11 04 47 AM](https://github.com/user-attachments/assets/26a01d4d-2dcd-4e0c-bd02-e6bc3bc3dd0a)

The prime directive of this migration is to have no content changes meaningful enough to have to go through any approval. A secondary directive is to try to have some parity with the guides, since they act as the same site.

@cantsin I'm guessing there are 1000+ _instances_ of this attribute going missing from the footer, but are there any other attributes that have gone missing?

**Comment by cantsin** [comment]
There are no other `aria-labels` missing, as far as I know. The fix indeed here is to add `aria-label` for the footer and yes, we should fix this on guides as well.

**Comment by cantsin** [comment]
Apologies for the noise, this aria-label is a misuse on 18f.gsa.gov 😞
